a08.Evaluator/Program.cs

```diff
@@ -18,4 +18,4 @@ class Program {
          Console.ResetColor ();

```

Comment:
Include 'using static System.Console;' at the top so that you can directly use it like this: ResetColor ()